GravatarApp/AppRoot/RootTabView.swift

```diff
@@ -4,12 +4,23 @@ import SwiftUI
 struct RootTabView: View {
     @StateObject private var avatarPickerViewModel: AvatarPickerViewModel
     @StateObject private var editProfileViewModel: EditProfileViewModel
+
+    let session: UserSession
+
     let onLogout: () -> Void
 
-    init(authToken: String, profile: Profile, onLogout: @escaping () -> Void) {
-        _avatarPickerViewModel = StateObject(wrappedValue: AvatarPickerViewModel(profile: profile, authToken: authToken))
-        _editProfileViewModel = StateObject(wrappedValue: EditProfileViewModel())
+    init(accessToken: String, profile: Profile, onLogout: @escaping () -> Void) {
+        let session = UserSession(profile: profile, accessToken: accessToken)
+        self.session = session
+
+        _avatarPickerViewModel = StateObject(wrappedValue: AvatarPickerViewModel(profile: profile, authToken: accessToken))
+        _editProfileViewModel = StateObject(wrappedValue: EditProfileViewModel(userSession: session))
         self.onLogout = onLogout
+
+        _avatarPickerViewModel = StateObject(
+            wrappedValue: AvatarPickerViewModel(profile: session.profile, authToken: session.accessToken)
+        )
+        _editProfileViewModel = StateObject(wrappedValue: EditProfileViewModel(userSession: session))
```

Comment:
Seems like these models are initialized twice by mistake, but no need to change anything here, I can remove them after merging this.